### PR TITLE
Additional animal icons

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #988, #1234, `crab` and `waves-shark-fin` in #1393, and `spider`/`-web` in #1495.

`beetle-scarab` matches `bug-*` icons in #988.

### CODE REVIEW

**Comment by jguddas** [comment]
- `bear-face`: cute but to big
- `butter-fly`: the antenna don't have 2px gap
- `cat-big`: not cute :/
- `cat`: I liked the old version
- `dog`: I liked the old version
- `frog`: hard to identify, I confused the nose holes with the eyes
- `owl`: big fan :owl:, maybe slightly big but pretty great anyway :+1: 
- `pig` & `piggy-bank`: I'm not a fan of the old version nor the changes

**Comment by karsa-mistmere** [comment]
I also prefer kemie's cat and dog icons, they're simpler and cleaner, yours aren't bad either, but I think the originals are a better fit for Lucide.

Also throwing in some animal icons from my drafts. Emphasis on drafts, some of these are pretty rough and/or have guideline violations:
image
https://www.figma.com/file/bCInvFWeG11jvpR14O7goR/lucide%2Fanimals?type=design&node-id=0%3A1&mode=design&t=vK5XOX8OFZ2q1gJu-1

File: icons/chameleon.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11 22c-5 0-9-4.5-9-10S6 2 11 2c2.2 0 4.2.9 5.7 2.3L19.3 2c3.1 3.1 3.5 7.9 1.3 11.4-.6.9-1.9.9-2.7.1l-1.2-1.2C15.2 10.9 13.2 10 11 10a6 6 0 0 0 0 12 4 4 0 0 0 0-8 2 2 0 0 0 0 4" />
```
Comment (jguddas) [inline icons/chameleon.svg:12]: Hard to identify as a chameleon.

File: icons/bee.svg [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m8 2 1.88 1.88" />
+  <path d="M14.1 3.88 16 2" />
+  <path d="M9 7V6a3 3 0 1 1 6 0v1" />
+  <path d="M5 7a3 3 0 1 0 2.2 5.1C9 10 12 7 12 7s3 3 4.8 5.1A3 3 0 1 0 19 7Z" />
+  <path d="M7.4 12h9.2" />
+  <path d="M6.9 17h10.2" />
+  <path d="M8.1 11.1c-2.1 2.1-2.1 5.6 0 7.8 0 0 1.9 2.1 3.9 3.1 2-1 3.9-3.1 3.9-3.1 2.1-2.1 2.1-5.6 0-7.8" />
```
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/bee.svg:18]: @karsa-mistmere I think one looks like a bee, the other a wasp 😅

File: icons/butterfly.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16 2c-4 2-4 7-4 7s0-5-4-7" />
+  <path d="M12 9v11" />
+  <path d="M20 5c-3.5 0-6.5 3.9-8 6.3C10.5 8.9 7.5 5 4 5a2 2 0 0 0-2 2c0 2.3.6 4.4 1.5 5.6C4 13.5 4.9 14 6 14h2c-.9.4-2.1.9-2.6 1.5-1.6 1.6-.9 3.4.7 4.9 1.6 1.6 3.4 2.3 4.9.7.3-.3 1-1.1 1-1.1s.6.8 1 1.1c1.6 1.6 3.4.9 4.9-.7 1.6-1.6 2.3-3.4.7-4.9-.5-.5-1.7-1.1-2.6-1.5h2c1.1 0 2-.5 2.5-1.4.9-1.2 1.5-3.3 1.5-5.6a2 2 0 0 0-2-2" />
```
Comment (karsa-mistmere) [inline icons/butterfly.svg:14]: Super cute, although the antennae are a bit dense at 100%.

On high dpi yours definitely looks better, still, maybe we should increase that gap by a little?
image
image

```suggestion
  <path d="M16 2a4 4 0 0 0-4 4 4 4 0 0 0-4-4" />
  <path d="M12 6v14" />
  <path d="M12 10c1.5-2.5 5.5-5 8-5a2 2 0 0 1 2 2c0 2.3-.6 4.4-1.5 5.6-.5.9-1.4 1.4-2.5 1.4h-2c.9.4 2.1 1 2.6 1.5 1.6 1.5.9 3.3-.7 4.9-1.5 1.6-3.3 2.3-4.9.7-.4-.3-1-1.1-1-1.1s-.7.8-1 1.1c-1.5 1.6-3.3.9-4.9-.7-1.6-1.5-2.3-3.3-.7-4.9.5-.6 1.7-1.1 2.6-1.5H6c-1.1 0-2-.5-2.5-1.4C2.6 11.4 2 9.3 2 7a2 2 0 0 1 2-2c2.5 0 6.5 2.5 8 5" />
```

File: icons/bee.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/bee.svg]: @karsa-mistmere Ok, I made him lose a little weight… Now uses the shape of https://lucide.dev/icons/eye—actually the eye from https://lucide.dev/icons/view, flipped 90°—for consistency…

[the remark this answers is not in the document]
Comment (karsa-mistmere) [reply icons/bee.svg]: I disagree, a wasp would look more like the third one:
image

File: icons/butterfly.svg (hunk 3 above)
Comment (jguddas) [reply icons/butterfly.svg]: Sneak peek of the low dpi preview, here with 18px which is commonly used in combination with text.



Looks pretty great IMO 💪

File: icons/chameleon.svg (hunk 1 above)
Comment (karsa-mistmere) [reply icons/chameleon.svg:12]: It's a really fun take on a Lucide-y chameleon, but I'm afraid @jguddas is right, it's just not unequivocally chameleon-like, _and_ it also violates some guidelines.

File: icons/bee.svg (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/bee.svg]: Would adding `bee` _and_ `wasp` not be overkill?

File: icons/butterfly.svg (hunk 3 above)
Comment (danielbayley) [reply icons/butterfly.svg]: > Sneak peek of the low dpi preview

@jguddas Nice, we definitely need to add this to the preview bot! Maybe I will get around to it eventually if no-one else does 😅

> On high dpi yours definitely looks better

@karsa-mistmere Shouldn’t we try to strike the right balance between high DPI—the vast majority of screens at this point—vs the lowest common denominator?

> maybe we should increase that gap by a little?

Yeah, although that might be a bit much; See latest `commit`… Aiming for somewhere in between. Again need that process to quickly see what it looks like at low res…

File: icons/horse-head.svg [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11.5 12H11" />
+  <path d="M5 15a4 4 0 0 0 4 4h7.8l.3.3a3 3 0 0 0 4-4.46L12 7c0-3-1-5-1-5S8 3 8 7c-4 1-6 3-6 3" />
+  <path d="M2 4.5C4 3 6 3 6 3l2 4" />
```
Comment (jguddas) [inline icons/horse-head.svg:14]:
```suggestion
```
icon - 2023-08-20T095638 796

File: icons/butterfly.svg (hunk 3 above)
Comment (karsa-mistmere) [reply icons/butterfly.svg]: > @karsa-mistmere Shouldn’t we try to strike the right balance between high DPI—the vast majority of screens at this point—vs the lowest common denominator?

[Citation needed.](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide)

Comment (danielbayley) [reply icons/butterfly.svg]: > [Citation needed.](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide)

@karsa-mistmere Interesting. I probably have a slightly privileged western bias… 😅

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Pull request merged**